crates/database/src/quotes.rs

```diff
@@ -281,7 +339,7 @@ mod tests {
         };
         assert_eq!(
             find(&mut db, &search_a).await.unwrap().unwrap(),
-            quotes_a[0],
```

Comment:
nit: let's avoid unnecessary changes to keep the diff small. Applies to the next 2 lines (and possibly some other spots as well).